### ISSUE TITLE
好奇一个问题

### ISSUE THREAD

**Issue by guoyingying432** [OlaWod/FreeVC#5]
您好，拜读了论文，想了解一下，模型的参数数量和运行时间大概是多久呢？或者说转换一秒的语音在3090显卡上需要多久的运算时间呢。

**Comment by OlaWod** [comment]
Number of parameters:
WavLM: 315.5 M
Other parts of model: 40.8 M

Inference speed:
On average it takes 0.05 seconds to generate 1 second speech. (tested with 4800 utterances for a total of 21336.72 seconds)

**Comment by guoyingying432** [comment]
多谢！另外同学我想再咨询一个问题，再原版VITS（https://github.com/jaywalnut310/vits/）中，在evaluate的过程中并没有计算在验证集上的loss，那该如何避免过拟合呢？如何选择最佳的模型呢？

**Comment by OlaWod** [comment]
> 多谢！另外同学我想再咨询一个问题，在原版VITS（[https://github.com/jaywalnut310/vits）中，在evaluate的过程中并没有计算在验证集上的loss，那该如何避免过拟合呢？如何选择最佳的模型呢？](https://github.com/jaywalnut310/vits%EF%BC%89%E4%B8%AD%EF%BC%8C%E5%9C%A8evaluate%E7%9A%84%E8%BF%87%E7%A8%8B%E4%B8%AD%E5%B9%B6%E6%B2%A1%E6%9C%89%E8%AE%A1%E7%AE%97%E5%9C%A8%E9%AA%8C%E8%AF%81%E9%9B%86%E4%B8%8A%E7%9A%84loss%EF%BC%8C%E9%82%A3%E8%AF%A5%E5%A6%82%E4%BD%95%E9%81%BF%E5%85%8D%E8%BF%87%E6%8B%9F%E5%90%88%E5%91%A2%EF%BC%9F%E5%A6%82%E4%BD%95%E9%80%89%E6%8B%A9%E6%9C%80%E4%BD%B3%E7%9A%84%E6%A8%A1%E5%9E%8B%E5%91%A2%EF%BC%9F)

The larger the dataset, the less likely it is to overfit.
A simple solution to choose the best checkpoint is to synthesize waveforms with different checkpoints and test their metrics (say, objective smos, l2 distance between $y$ and $y_{hat}$, etc.) results. We can also log the metrics results of validation set during validation, just 

```python
def evaluate(...)
    ...
    y_hat, ... = generator.module.infer(...)
    foo = metrics1(y_hat, ...)
    bar = metrics2(y_hat, ...)
    ...
    scalar_dict.update({"metrics1": foo, "metrics2": bar})
    ...
```
